tests/ui/conftest.py

```diff
@@ -77,10 +69,17 @@ def selenium(selenium, request):
 
 
 @pytest.fixture
-def devhub_login(selenium, fxa_account):
+def fxa_account():
+    setattr(fxa_account, 'email', os.environ['FXA_EMAIL'])
+    setattr(fxa_account, 'password', os.environ['FXA_PASSWORD'])
```

Comment:
What's wrong with doing

```python
fxa_account.email = os.environ['FXA_EMAIL']
fxa_account.password = os.environ['FXA_PASSWORD']
```

Also, we should fail in a reasonable way in case `FXA_EMAIL` and `FXA_PASSWORD` are not set in the environment.

Reply:
`setattr` doesn't do what i think you think it does. The value you're passing in isn't a default value, it's the value this object gets set to.

```
Signature: setattr(obj, name, value)
Docstring:
Sets the named attribute on the given object to the specified value.

setattr(x, 'y', v) is equivalent to ``x.y = v''
```

It's literally the same.

Reply:
Oh whoops, I may have been thinking of something else.